docker/base-images/base-admin-tools.Dockerfile

```diff
@@ -8,9 +23,15 @@ RUN apk add --update --no-cache \
     jq \
     mysql-client \
     postgresql-client \
-    py-pip \
+    py3-pip \
     expat \
-    tini \
-    && pip install cqlsh
+    tini
+
+COPY --from=builder /opt/venv /opt/venv
+
+ENV PATH="/opt/venv/bin:$PATH"
+
+# validate installation
+RUN cqlsh --version
```

Comment:
Oh I meant outside of the build to avoid creating an extra layer, address at will.